eng/pipelines/templates/stages/archetype-python-release.yml

```diff
@@ -96,23 +96,34 @@ stages:
                           scriptPath: 'scripts/devops_tasks/verify_dependencies_present.py'
                           arguments: '--package-name ${{ artifact.name }} --service ${{ parameters.ServiceDirectory }}'
 
-                      - task: TwineAuthenticate@1
-                        displayName: 'Authenticate to registry: pypi.org'
-                        inputs:
-                          pythonUploadServiceConnection: 'PyPI Upload (azure-sdk token)'
-
                       - task: TwineAuthenticate@0
                         displayName: 'Authenticate to feed: ${{parameters.DevFeedName}}'
                         inputs:
                           artifactFeeds: ${{parameters.DevFeedName}}
 
-                      - script: |
-                          set -e
-                          twine upload --repository 'pypi' --config-file $(PYPIRC_PATH) $(Pipeline.Workspace)/${{parameters.ArtifactName}}/${{artifact.name}}/*.whl
-                          echo "Uploaded whl to pypi"
-                          twine upload --repository 'pypi' --config-file $(PYPIRC_PATH) $(Pipeline.Workspace)/${{parameters.ArtifactName}}/${{artifact.name}}/*.zip
-                          echo "Uploaded zip to pypi"
-                        displayName: 'Publish package to registry: pypi.org'
+                      - pwsh: |
+                          $esrpDirectory = "$(Pipeline.Workspace)/esrp-release/${{parameters.ArtifactName}}/${{artifact.name}}"
+                          New-Item -ItemType Directory -Force -Path $esrpDirectory
+
+                          Get-ChildItem -Path "$(Pipeline.Workspace)/${{parameters.ArtifactName}}/${{artifact.name}}" `
+                            | Where-Object { ($_.Name -like "*.zip" -or $_.Name -like "*.whl") } `
+                            | Copy-Item -Destination $esrpDirectory
+
+                          Get-ChildItem $esrpDirectory
+                        displayName: Isolate files for ESRP Publish
+
+                      - task: EsrpRelease@2
+                        displayName: 'Publish to ESRP'
+                        inputs:
+                          ConnectedServiceName: 'ESRP Release Service'
+                          Intent: 'PackageDistribution'
+                          ContentType: 'PyPI'
+                          PackageLocation: $(Pipeline.Workspace)/esrp-release/${{parameters.ArtifactName}}/${{artifact.name}}
+                          Owners: $(Build.RequestedForEmail)
+                          Approvers: 'wesh@microsoft.com'
```

Comment:
I know you just copied this but we should probably change this to a more generic alias either our team or bot name. Or mabye we can see if this is actually required at all.

Reply:
The "approvers" field here? Yeah I'll ask Akoly.

Reply:
Ok looks likeit can be any single email. Going to swap this to the RequestedForEmail.